Packs/SecurityTrails/Integrations/SecurityTrails/SecurityTrails.py

```diff
@@ -0,0 +1,1069 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+''' CONSTANTS '''
+
+removed_keys = ['endpoint', 'domain', 'hostname']
+
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+
+    def handle_error(self, message: dict = {}):
+        return_error(message['message'])
+
+    def domain_tags(self, hostname: str = None):
+        res = self._http_request(
+            'GET',
+            f'domain/{hostname}/tags',
+            ok_codes=[200, 403],
+            resp_type="response"
+        )
+        if res.status_code != 200:
+            self.handle_error(res.json())
```

Comment:
This also being taken care of in the `_http_request` function.

Reply:
Again, this is handled by the _http_request, but, this error handler handles the "error" message in the returned JSON which is easier to read by the user.